Main/Source/gods.cpp

```diff
@@ -1223,27 +1392,30 @@ void cruentus::PrayGoodEffect()
            && !Audience->TemporaryStateIsActivated(PANIC)
            && PLAYER->GetRelation(Audience) == HOSTILE
            && Audience->GetPanicLevel() > RAND() % 33)
```

Comment:
It might pay to add a check for player wisdom in these conditionals to make wiser players have better chances...